src/rgw/rgw_civetweb_frontend.cc

```diff
@@ -68,9 +72,22 @@ int RGWCivetWebFrontend::process(struct mg_connection*  const conn)
   RGWRequest req(env.store->getRados()->get_new_req_id());
   int http_ret = 0;
   //assert (scheduler != nullptr);
-  int ret = process_request(env.store, env.rest, &req, env.uri_prefix,
-                            *env.auth_registry, &client_io, env.olog,
-                            null_yield, scheduler.get() ,&http_ret);
+  int ret;
+  #ifdef WITH_JAEGER
```

Comment:
instead of having function definition changed everywhere with 2 alternatives, can you try testing adding span to RGWSimpleRequest https://github.com/ideepika/ceph/blob/jaeger-rebase-8.0/src/rgw/rgw_rest_client.h#L10:L10